manifest.php

```diff
@@ -34,7 +34,7 @@
     'label' => 'Test core extension',
 	'description' => 'TAO Tests extension contains the abstraction of the test-runners, but requires an implementation in order to be able to run tests',
     'license' => 'GPL-2.0',
-    'version' => '3.1.0',
+    'version' => '3.1.1',
```

Comment:
As it is a feature, it should be a minor version change, not a patch version change:
`3.1.0` => `3.2.0`